app/controllers/client2controller.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+class ClientsController extends \BaseController {
+
+	/**
+	 * Display a listing of the resource.
+	 * GET /clients
+	 *
+	 * @return Response
+	 */
+	public function index()
+	{	
+		$pTitle		= "Clients";
+
+		$id 		= Auth::id();
+		$user		= User::find($id);
+		$clients 	= $user->clients()->orderBy('created_at', 'desc')->take(5)->get();
+
+		$counter 	= 0;		
+		return View::make('clients.index', compact([ 'clients', 'counter', 'pTitle']));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "pTitle" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://172.27.59.144:9000/coding_rules#rule_key=php%3AS1192)